chain/network/src/types.rs

```diff
@@ -435,71 +395,17 @@ pub enum NetworkResponses {
     RouteNotFound,
 }
 
-#[derive(actix::Message, Debug, strum::AsRefStr, strum::IntoStaticStr)]
-// TODO(#1313): Use Box
-#[allow(clippy::large_enum_variant)]
-#[rtype(result = "NetworkClientResponses")]
-pub enum NetworkClientMessages {
```

Comment:
Similarly, not moving the home of this message would've reduced the diff substantially

Reply:
done